go.mod

```diff
@@ -3,50 +3,22 @@ module github.com/makerdao/vulcanizedb
 go 1.12
 
 require (
-	github.com/allegro/bigcache v1.2.1 // indirect
-	github.com/apilayer/freegeoip v3.5.0+incompatible // indirect
-	github.com/aristanetworks/goarista v0.0.0-20191106175434-873d404c7f40 // indirect
-	github.com/btcsuite/btcd v0.20.1-beta // indirect
-	github.com/cespare/cp v1.1.1 // indirect
 	github.com/dave/jennifer v1.3.0
-	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/docker/docker v1.13.1 // indirect
-	github.com/edsrzf/mmap-go v1.0.0 // indirect
-	github.com/elastic/gosigar v0.10.5 // indirect
-	github.com/ethereum/go-ethereum v1.9.6
-	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
-	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08 // indirect
-	github.com/gorilla/websocket v1.4.1 // indirect
+	github.com/ethereum/go-ethereum v1.9.8
```

Comment:
If we're bumping this, might be cool to bump it even higher - but idk if it matters given the `replace` below 🤔